docs/bridge/docs/03-RFQ/index.md

```diff
@@ -0,0 +1,209 @@
+[Quoter]: /docs/RFQ/#entities
+[Prover]: /docs/RFQ/#entities
+[Relayer]: /docs/RFQ/#entities
+[Guard]: /docs/RFQ/#entities
+[Canceler]: /docs/RFQ/#entities
+
+# Synapse RFQ Summary
+
+Synapse RFQ (Request-For-Quote) is an <abbr title="'Intent' refers to a user authorizing specific actions that they want to achieve, typically in very simple terms, such as a bridge or swap. Actual execution of the actions is then performed on the user's behalf by third parties known as solvers/relayers.">intent</abbr>-based bridging system that connects bridging users to a network of relayers.
+
+These relayers compete to provide the optimal bridge execution (eg: the best price) for the user's specific request.
+
+<h2 style={{ textAlign: 'center' }}>Request-For-Quote End-to-End Flow</h2>
+<figure>
+    <RFQFlow />
+</figure>
+<br />
+<div style={{ display: 'flex', justifyContent: 'center' }}>
+  <table>
+    <tbody>
+      <tr>
+        <td><strong>Order</strong></td>
+        <td>User inputs their desired order into a bridge interface.</td>
+      </tr>
+      <tr>
+        <td><strong>Quote</strong></td>
+        <td>Quotes from Relayers are evaluated and resolved to the optimal choice to match the user's input.
+        <br/><br/>The resolved quote is used to construct a bridge transaction for the user to sign.</td>
+      </tr>
+      <tr>
+        <td><strong>Request</strong></td>
+        <td>User signs and submits the bridge transaction on-chain.
+        <br/><br/>Their assets are deposited into a Bridge contract on the `originChain`.</td>
+      </tr>
+      <tr>
+        <td><strong>Relay</strong></td>
+        <td>A [Relayer] completes the user's bridge on the `destChain` by delivering the user's desired funds.
+        <br/><br/>At this point, the bridge is complete from the user's perspective.</td>
+      </tr>
+      <tr>
+        <td><strong>Prove</strong></td>
+        <td>The [Relayer] submits an optimistic proof of their completed relay to the `originChain`.</td>
+      </tr>
+      <tr>
+        <td><strong>Guard</strong></td>
+        <td>After a proof is asserted for each bridge, a brief optimistic dispute period is initiated.
+        <br/><br/>During this period, [Guard] entities will verify that the proof is valid and accurate.</td>
+      </tr>
+      <tr>
+        <td><strong>Claim</strong></td>
+        <td>After the dispute period has passed, the [Relayer] may claim the user's originally deposited funds.
+        <br/><br/>This reimburses the [Relayer] for the funds which they delivered in the earlier [relay] step.</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+## Entities
+
+<b>Users</b> <span style={{color: 'darkgray'}}><i>(Permissionless Role)</i></span>
+    <blockquote>
+        Uses a bridge interface to submit their intent to the RFQ system for a quote, and then to the chain for fulfillment.
+    </blockquote><br/>
+
+<b>Quoters</b> <span style={{color: 'darkgray'}}><i>(Permissioned Role)</i></span>
```

Comment:
This is currently permissioned (and will remain to an extent even if Quoters need to post a collateral in order to be registered).

Reply:
It does say that Quoters are a permissioned role - am I misunderstanding your comment?

Reply:
Strange, I guess I've managed to read this wrong. This is indeed correct :)